util/botStatsHandler.js

```diff
@@ -0,0 +1,21 @@
+const { post } = require("snekfetch");
+
+module.exports = class BotListHandler {
+  constructor(client) {
+    this.client = client;
+    this.interval = this.client.setInterval(this.updateStats.bind(this), 9e5);
```

Comment:
`9e5` is an unusual number. Can you document what does that represent (in minutes or hours)?